src/Persistent.hs

```diff
@@ -0,0 +1,12 @@
+{-# OPTIONS_GHC -Werror #-}
+
+module Persistent (module Api.Persistent, runPersist)
+where
```

Comment:
I wanted to separate the active persistent handler from the API of Persistent. It had to be placed into a different module to avoid a circular dependency.